iroh/src/endpoint.rs

```diff
@@ -1286,13 +1339,38 @@ impl Future for Connecting {
             Poll::Ready(Err(err)) => Poll::Ready(Err(err)),
             Poll::Ready(Ok(inner)) => {
                 let conn = Connection { inner };
-                try_send_rtt_msg(&conn, this.ep);
+                try_send_rtt_msg(&conn, this.ep, *this.remote_node_id);
                 Poll::Ready(Ok(conn))
             }
         }
     }
 }
 
+/// Future that completes when a connection is fully established
```

Comment:
```suggestion
/// Future that completes when a connection is fully established.
```

We follow the rust style guide, not the Quinn style guide.  I know, it's terrible.